specs/APIReview_CustomDownload.md

```diff
@@ -0,0 +1,346 @@
+# Background
+<!-- TEMPLATE
+    Use this section to provide background context for the new API(s)
+    in this spec.
+
+    This section and the appendix are the only sections that likely
+    do not get copied into any official documentation, they're just an aid
+    to reading this spec.
+
+    If you're modifying an existing API, included a link here to the
+    existing page(s) or spec documentation.
+
+    For example, this section is a place to explain why you're adding this
+    API rather than modifying an existing API.
+
+    For example, this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...".
+-->
+There have been multiple requests by WebView Developers to be able to customize/control the download experience within WebView2.
+
+Some of the requests we've heard, and want to solve with this API are:
+- Developers want to be able to disable a download for the entire environment.
+- Developers want to be able to disable downloads based on a per-case basis. (Content Type, User).
+- Developers want to have access to downloaded file’s metadata, and be able to prevent downloads based on this information.
+   Ex. Dev wants to block downloads above 1GB.
+- Developers want to modify the download path.
+- Developer don’t want to see edge download UI if download is cancelled.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+
+Our proposed solution is to give developers an API to control the download experience.
+We will do this by exposing a DownloadStarting event, as well as a DownloadStartingEventHandler.
```

Comment:
Probably just skip the ", as well as a " part, since this is COM specific. The DownloadStarting event is common to all languages and implies the event handler interface for the COM API.